Sources/HTTP/Server/ServerError.swift

```diff
@@ -6,4 +6,5 @@ public enum ServerError: Swift.Error {
     case respond(Swift.Error)
     case dispatch(Swift.Error)
     case unknown(Swift.Error)
+    case notSupported
```

Comment:
We'll have to pull this out unfortunately. We can throw a `respond` error and have a `notSupported` error somewhere else for API preservation.